README.adoc

```diff
@@ -0,0 +1,5 @@
+# Eclipse MicroProfile
+
+This repository is no longer the active process for submitting feature ideas for Eclipse MicroProfile.
+
```

Comment:
Fair enough.  Maybe that could be part of the process is to ensure that the Proposal is kept as part of their eventual repo.  Establishing a wiki page to keep track of the proposals and repos might be a nice reference.  I'm just thinking that we don't want prospective users to have to know where to look (or hunt) to track down this information.